dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/search/SearchServiceBean.java

```diff
@@ -596,7 +596,7 @@ public SolrQueryResponse search(DataverseRequest dataverseRequest, List<Datavers
         boolean draftsAvailable = false;
         boolean unpublishedAvailable = false;
         boolean deaccessionedAvailable = false;
-        boolean hideMetadataSourceFacet = true;
+        boolean hideMetadataSourceFacet = false;
```

Comment:
do we want to change default behaviour? Now metadata source will be always present.

Reply:
I wasn't sure. On one hand it's a change in app logic, on the other it's sort of hiding functionalities and we have agreed not to do that on development environment. Probably it's best not to change the code here.